src/app/chatroom/chatroom.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
Either add tests or delete this file, the auto-generated one does not add value.